AWSS3/AWSS3TransferUtility.m

```diff
@@ -416,7 +414,7 @@ - (instancetype)initWithConfiguration:(AWSServiceConfiguration *)serviceConfigur
         }
         
         if (identifier) {
-            _sessionIdentifier = identifier;
+            _sessionIdentifier = [NSString stringWithFormat:@"%@.%@", AWSS3TransferUtilityDefaultIdentifier, identifier];
```

Comment:
This has the potential to break clients who are calling the init function.  Thanks @palpatim